yksgi/src/llvmbridge.rs

```diff
@@ -169,6 +192,10 @@ impl Value {
         unsafe { Type(LLVMTypeOf(self.0)) }
     }
 
+    pub fn get_ffi_type(&self) -> FFIType {
```

Comment:
Is it cleaner if we do `ret_val.get_type().ffi_type()` instead of having this extra function here?

Reply:
I prefer to have the helper to make the consumers more concise.

Reply:
I just feel like `get_ffi_type` shouldn't be a function on a `Value` as it solely relates to the type. Adding 3 lines to save 7 characters (`_type()`) looks like a net loss to me. ;-)

Reply:
Fixed in ce3da0363